### PR TITLE
CIE-3997.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/configuration-bundle/configuring-cpu-memory-for-opcua-apama.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   - /etc/init.d/opcua-device-gateway
+   - /etc/init.d/apama
+   - /etc/init.d/opcua-mgmt-service
+3. Edit the parameter `CTRL_DOCKER_OPTIONS` to the required values.
+4. Restart the Apama and OPCUA services:
+   - `[admin@iot-edge-server ~]$ sudo service apama restart`
+   - `[admin@iot-edge-server ~]$ sudo service opcua-mgmt-service restart`
+   - `[admin@iot-edge-server ~]$ sudo service opcua-device-gateway restart`
+
+>**Important:** Since the changes to this file are overwritten when the Edge appliance is updated, you must reapply the changes to this file after the update process.
```
[the remark this answers is not in the document]
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-cpu-memory-for-opcua-apama.md]: Done

**Review by BeateRixen — APPROVED** [review]

File: content/edge/configuration-bundle/configuring-apama.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+weight: 52
+title: Configuring Apama
+layout: redirect
+---
+
+The memory limit for Apama container is set to 2048 MB and the CPU limit is set to 256 as default values.
+
+`CTRL_DOCKER_OPTIONS="--memory 2048m --cpu-shares 256"`
```
Comment (bbyreddy) [inline content/edge/configuration-bundle/configuring-apama.md:9]: - [ ] We will have to explain the unit CPU Shares here. And also it is not necessary that the user provides the CPU limits in CPU Shares. CPU Shares actually define the share or percentage of total CPUs allocated to the VM. But there is another unit, through which one can provide the absolute value of the CPU limit. So, explaining these options and units or providing the use a reference would help.

File: content/edge/configuration-bundle/configuring-apama.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+weight: 52
+title: Configuring Apama
+layout: redirect
+---
+
```
Comment (bbyreddy) [inline content/edge/configuration-bundle/configuring-apama.md:6]: - [ ] I understand that "Configuring Apama" is the new section. But within this section, we can include multiple sub sections right? What is the subsection heading for changing the memory and CPU configurations. In my opinion, we should follow the Karaf's pattern here.

**Review by bbyreddy — COMMENTED** [review]
@sharath-jk minor comments.

**Review by MWindel — APPROVED** [review]

File: content/edge/configuration-bundle/configuring-apama.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-apama.md:11]: Done.

File: content/edge/configuration-bundle/configuring-apama.md (hunk 3 above)
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-apama.md:6]: Done.

File: content/edge/configuration-bundle/configuring-apama.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+### Updating the memory and CPU limit for Apama
+
+If the system performance is slow, you must increase the memory. The memory limit for Apama container is set to 2048 MB and the CPU limit is set to 256 as default values. For most use cases, these values are sufficient. You can change the values as per your requirements.
```
Comment (bbyreddy) [inline content/edge/configuration-bundle/configuring-apama.md:9]: - [ ] I think it is better to be specific about which system we are referring to. "If the Apama's performance is slow...." 
- [ ] Or we can just start this para with "The memory limit for Apama container is set...."

File: content/edge/configuration-bundle/configuring-opcua.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+weight: 51
+title: Configuring OPCUA
+layout: redirect
+---
+
+### Updating the memory and CPU limit for OPCUA
+
+If the system performance is slow, you must increase the memory. The memory limit for OPCUA container is set to 2048 MB and the CPU limit is set to 256 as default values. For most use cases, these values are sufficient. You can change the values as per your requirements.
```
Comment (bbyreddy) [inline content/edge/configuration-bundle/configuring-opcua.md:9]: - [ ] I think it is better to be specific about which system we are referring to. "If the OPCUA's performance is slow...."
- [ ] Or we can just start this para with "The memory limit for OPCUA containers is set...."

File: content/edge/configuration-bundle/configuring-apama.md (hunk 4 above)
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-apama.md:9]: Done. I have deleted the lead-in sentence.

File: content/edge/configuration-bundle/configuring-opcua.md (hunk 5 above)
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-opcua.md:9]: Done. I have deleted the lead-in sentence.

**Review by bbyreddy — APPROVED** [review]

**Pull request merged**